packages/ui/src/Alert.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { AnchorHTMLAttributes, FC, ReactNode } from 'react'
+import cn from 'classnames'
+import { X, ChevronRight } from 'react-feather'
+
+import { PartialRequired } from '@hazelcast/helpers'
+
+import { Link } from './Link'
+import { Button } from './Button'
+import { ToastIcon, ToastType } from './Toast'
+import { Icon, IconProps } from './Icon'
+import { IconButton } from './IconButton'
+
+import styles from './Alert.module.scss'
+
+type AccessibleActionButtonIconProps =
+  | {
+      icon: IconProps['icon']
+      iconAriaLabel: string
+    }
+  | {
+      icon?: never
+      iconAriaLabel?: never
+    }
+
+export type AlertActionButton = {
+  text: string
+  onClick: (e: React.MouseEvent<HTMLButtonElement>) => void
+} & AccessibleActionButtonIconProps
+
+export type AlertActionLink = {
+  text: string
+} & PartialRequired<AnchorHTMLAttributes<HTMLAnchorElement>, 'href'>
+
+export type AlertAction = AlertActionButton | AlertActionLink
+
+const isAlertActionButton = (action: AlertAction): action is AlertActionButton => (action as AlertActionButton).onClick !== undefined
+
+export type AlertProps = {
+  type: ToastType
+  title: string
+  content: ReactNode
+  actions?: AlertAction[]
+  className?: string
+  closeToast: (e: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+export const Alert: FC<AlertProps> = ({ type, title, content, actions, className, closeToast }) => {
+  const { icon, ariaLabel } = ToastIcon[type]
+
+  return (
+    <div
+      data-test="alert"
+      className={cn(className, styles.alert, {
+        [styles.success]: type === 'success',
+        [styles.info]: type === 'info',
+        [styles.warning]: type === 'warning',
+        [styles.critical]: type === 'critical',
+      })}
+    >
+      <div className={styles.header}>
+        <Icon data-test="alert-icon" ariaLabel={ariaLabel} icon={icon} className={styles.icon} />
+        <div data-test="alert-title" className={styles.title}>
+          {title}
+        </div>
+        <IconButton
```

Comment:
I am wondering if we should add `transparent` type here as well. We also need to think if `#FAFAFA` background works as the hover effect. @pawelklasa WDYT?
The current design of Icon Button (https://www.figma.com/file/8mVm6LTbp2Z0RaWWjTZoft/%F0%9F%90%9DHIVE---Hazelcast-Design-System?node-id=3352%3A8) does not play nice with different background colors.
We would like to use it for the `X` button for our alerts https://www.figma.com/file/8mVm6LTbp2Z0RaWWjTZoft/%F0%9F%90%9DHIVE---Hazelcast-Design-System?node-id=2341%3A111, but the `#FAFAFA` hover does not fit there. Should we do something else for the hover effect? Maybe, we could change the color instead? E.g. for the hover. of the `neutral-white` Icon Button we could make it `neutral0lighter`, for the `colot-text` we could do `secondary-text`. Alternatively, we could darker the background. WDYT?

Reply:
I would lean towards changing the `IconButton` hover/rollover. If we can find an accessible alternative, to this `background-color` toggle, great! If not, we can surely add `transparent` `IconButton` type. This is makes sense to me, as we already implement it for regular button, right?

I'm a. bit lost in the last couple of sentences, though.

> Maybe, we could change the color instead? E.g. for the hover. of the neutral-white Icon Button we could make it neutral0lighter, for the colot-text we could do secondary-text. Alternatively, we could darker the background. WDYT?

`IconButton` does not have text, right?

Reply:
@jankoritak I meant that for the hover effect we could simply darken the background or darken the icon color.

Reply:
I see now. Let's wait for @pawelklasa's POV.